.teamcity/src/subprojects/eap/ProjectPublishEAPToSpace.kt

```diff
@@ -0,0 +1,43 @@
+package subprojects.eap
+
+import jetbrains.buildServer.configs.kotlin.v2019_2.*
+import subprojects.*
+import subprojects.build.*
+import subprojects.build.core.*
+import subprojects.release.*
+import subprojects.release.publishing.*
+
+object ProjectPublishEAPToSpace : Project( {
+    id("ProjectKtorPublishEAPToSpace")
+    name = "Release Ktor EAP"
+    description = "Publish on a nightly basis EAP branches to Space"
+
+    params {
+        defaultTimeouts()
+        param("env.SIGN_KEY_ID", value = "")
```

Comment:
I'm not sure that it will work with check in the `publish.gradle`:
```kotlin
if (signingKey != null) { }
```
We should either check for empty string in `publish.gradle` or remove this param at all.